frontend/src/styles/CaptionVotes.css

```diff
@@ -26,4 +26,14 @@
 
 .vote-ticker:hover{
```

Comment:
Did you forget to change the class name here?

Reply:
Good catch, thanks